.hie-bios

```diff
@@ -1,2 +1,29 @@
-#!/bin/sh
-cat "$HIE_BIOS_ARGS" | sed "s/'//g" | sed "s/-qg//g" | sed "s/+RTS.*-RTS//" | sed "s|/bazel-out|bazel-out|g" | sed "s/-fexternal-interpreter//g" | tr " " "\n" > "$HIE_BIOS_OUTPUT"
+#!/usr/bin/env bash
+set -euo pipefail
+hie_bios_flags() {
+  bazel build //compiler/damlc:damlc@ghci \
+    --output_groups=hie_bios \
+    --experimental_show_artifacts \
+    2>&1 \
+    | awk '
+      BEGIN { rts_opts = false }
```

Comment:
:crying_cat_face: I wonder if `hie-bios` should make it possible to reuse the logic for flag filtering even if you define your own `.hie-bios` file.